tests/standalone/six.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+import six
+import sys
+
+MAXSIZE = sys.maxsize
+
+def test_dispatch_types(value):
```

Comment:
The Nuitka test runner is executing these as programs, with CPython and Nuitka, and will check the outputs, and merely defining a function named `test_*` is not going to do anything to run it. You can put stuff in functions, but then you have to also call it. Most often for standalone tests, we do just put things on module level.